pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/data/BrokerLookupDataTest.java

```diff
@@ -44,21 +45,21 @@ public void testConstructors() {
                 webServiceUrl, webServiceUrlTls, pulsarServiceUrl,
                 pulsarServiceUrlTls, advertisedListeners, protocols, true, true,
                 ExtensibleLoadManagerImpl.class.getName(), System.currentTimeMillis(),"3.0");
-        Assert.assertEquals(webServiceUrl, lookupData.webServiceUrl());
-        Assert.assertEquals(webServiceUrlTls, lookupData.webServiceUrlTls());
-        Assert.assertEquals(pulsarServiceUrl, lookupData.pulsarServiceUrl());
-        Assert.assertEquals(pulsarServiceUrlTls, lookupData.pulsarServiceUrlTls());
-        Assert.assertEquals(Optional.of("9092"), lookupData.getProtocol("kafka"));
-        Assert.assertEquals(Optional.empty(), lookupData.getProtocol("echo"));
-        Assert.assertTrue(lookupData.persistentTopicsEnabled());
-        Assert.assertTrue(lookupData.nonPersistentTopicsEnabled());
-        Assert.assertEquals("3.0", lookupData.brokerVersion());
+        assertEquals(webServiceUrl, lookupData.webServiceUrl());
+        assertEquals(webServiceUrlTls, lookupData.webServiceUrlTls());
```

Comment:
IIRC the order of actual and expected are inverted between junit and testng. Could you check if the order is correct now?

Reply:
The same question should be asked and answered above.